_data/mentors.yml

```diff
@@ -1836,7 +1837,8 @@
 - name: Lidiane Mayra Taquehara
   disabled: false
   matched: false
-  sort: 100
+  sort: 500
```

Comment:
Please add screenshot of the application not the code. I am not sure it is working.